### DISCUSSION TITLE
How to connect Hue Essentials to HA addon

### DISCUSSION THREAD

**Discussion by Burn-In** [maxbec/hassio-diyHue#10] [Q&A]
I have the addon running in HA, the webinterface works. But how can i connect Hue Essentials to it to add some devices?

**Comment by maxbec** [comment]
Hello @Burn-In,

thank you for the question.
In Hue Essentials just select "Manual" -> Then select diyhue and enter the IP-Adress of your Hassio instance.
That should work in other Apps as well.

**Comment by Burn-In** [comment]
Oh yes. This was my mistake. I should have work the whole time. The IP address has changed after the HA installation. Thank you.
